images/router/clear-route-status.sh

```diff
@@ -84,8 +84,15 @@ if [[ ${#} -ne 2 || "${@}" == *" help "* ]]; then
 fi
 
 if ! command -v jq >/dev/null 2>&1; then
-    printf "%s\n%s\n" "Command line JSON processor 'jq' not found." "please install 'jq' to use this script."
-    exit 1
+    printf "%s\n%s\n" "Command line JSON processor 'jq' not found." "please install 'jq' version greater than 1.4 to use this script."
+    exit
+fi
+
+jq_present=1
+echo '' | jq '."status"."ingress"|=map(select(.routerName != "test"))' >/dev/null 2>&1 || jq_present=0
+if [[ "${jq_present}" -eq 0 ]]; then
```

Comment:
Nit, but this is a little obfuscated as written... simpler:

```sh
if ! jq '."status"."ingress"|=map(select(.routerName != "test"))' <<<"" >/dev/null 2>&1; then;
```

Or:

```sh
if ! echo | jq '."status"."ingress"|=map(select(.routerName != "test"))' >/dev/null 2>&1; then;
```

I'm still worried that the `>/dev/null 2>&1` may cause this to fail without a trace of _why_ and lead to a situation where it's hard to debug, but if that worry is not echoed by you all then it can be ignored.